scripts/simple.py

```diff
@@ -0,0 +1,7 @@
+def uftrace_entry(args):
```

Comment:
I think we don't have to put `#!/usr/bin/env python` because it's for shell to understand the script with python interpreter.